zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/CustomListCollector.java

```diff
@@ -25,7 +25,11 @@
 
     @Override
     public BiConsumer<ImmutableList.Builder<T>, T> accumulator() {
-        return (builder, item) -> builder.add(item);
+        return (builder, item) -> {
+            if (item != null) {
+                builder.add(item);
+            }
+        };
```

Comment:
Cannot see where this is being used. Just double check its required and does not break anything unexpected.